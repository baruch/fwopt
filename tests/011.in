iptables -A INPUT -s 192.122.181.0/24 -j ACCEPT
iptables -A FORWARD -d 15.0.0.0/255.255.0.0 -j ACCEPT
iptables -A OUTPUT --dst 16.1.2.3 -j ACCEPT
