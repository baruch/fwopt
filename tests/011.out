iptables -A INPUT --src 192.122.181.0/255.255.255.0 -j ACCEPT
iptables -A INPUT --dst 16.1.2.3 -j DROP
iptables -A OUTPUT --dst 16.1.2.3 -j ACCEPT
iptables -A FORWARD --dst 15.0.0.0/255.255.0.0 -j ACCEPT
